concent_api/api-integration-multiprocess.py

```diff
@@ -0,0 +1,92 @@
+#!/usr/bin/env python3
+
+import hashlib
+import os
+import sys
+import time
+from datetime import datetime
+from threading import Thread
+
+import requests
+from golem_messages.shortcuts import dump
+
+from api_testing_common import api_request
+from api_testing_common import get_force_get_task_result
+from api_testing_common import REQUESTOR_PRIVATE_KEY
+from api_testing_common import run_tests
+from common.helpers import get_current_utc_timestamp
+
+os.environ.setdefault("DJANGO_SETTINGS_MODULE", "concent_api.settings")
+
+now = datetime.now()
+testing_subtask_id = str(now.year) + str(now.month) + str(now.day) + str(now.hour) + str(now.minute) + str(
+    now.second) + str(now.microsecond)
+
+responses = []
+
+
+def multiprocess(func):
+    def wrapper(*args, **kwargs):
+        for i in range(3):
+            t = Thread(target=func, args=(*args,), kwargs={**kwargs})
+            t.start()
+
+    return wrapper
+
+
+def test_case_that_multiple_requests_by_one_subtask_will_not_be_cause_of_server_error(cluster_consts, cluster_url, test_id):
+
+    send_correct_request(cluster_consts=cluster_consts, cluster_url=cluster_url)
+
+    time.sleep(4)
+    print('Responses = ' + str(responses))
+    assert 'AckForceGetTaskResult' in responses
+    responses.remove('AckForceGetTaskResult')
```

Comment:
why would you remove anything from `responses`?

Reply:
change to compare the whole list. `AckForceGetTaskResult` should be always first, so i can simplify check the responses